lookout/style/format/benchmarks/quality_report_noisy.py

```diff
@@ -305,45 +301,50 @@ def quality_report_noisy(bblfsh: str, language: str, confidence_threshold: float
-           Should be strings separated by newlines.
-    :param retrain: Set True if you want to force model retraining.
+    :param repos: Input list of urls to the repositories to analyze. \
+           Should be strings separated by newlines. If if is None, \
+           we use the string defined at the beginning of the file.
     """
     log = logging.getLogger("quality_report_noisy")
+
+    # initialization
     repo_names = []
     last_accepted_rule = {}
-    precisions, recalls, accepted_rules = (defaultdict(list) for _ in range(3))
-    n_mistakes, prec_max_rec, confidence_threshold_exp, max_rec, \
+    predrs, precisions, accepted_rules = (defaultdict(list) for _ in range(3))
+    n_mistakes, prec_max_predr, confidence_threshold_exp, max_predr, \
         n_rules, n_rules_filtered = ({} for _ in range(6))
     if repos is None:
         repos = REPOSITORIES
     try:
+        # fetch the the original and noisy repositories
         client = BblfshClient(bblfsh)
         log.info("Repositories: %s", repos)
         with tempfile.TemporaryDirectory() as tmpdirname:
             for raw in repos.splitlines():
                 repo_path, clean_commit, noisy_commit = raw.split(",")
                 repo = repo_path.split("/")[-1]
-                if repo_path.startswith("https://github.com"):
-                    log.info("Fetching %s", repo_path)
-                    git_dir = os.path.join(tmpdirname, repo)
-                    git_dir_noisy = os.path.join(tmpdirname, repo + "_noisy")
-                    cmd1 = "git clone --single-branch --branch master %s %s" % (repo_path, git_dir)
-                    cmd2 = "git clone --single-branch --branch style-noise-1-per-file %s %s" \
-                        % (repo_path, git_dir_noisy)
-                    try:
-                        for cmd in (cmd1, cmd2):
-                            log.debug("Running: %s", cmd)
-                            subprocess.check_call(cmd.split())
-                    except subprocess.CalledProcessError as e:
-                        raise ConnectionError("Unable to fetch repository %s" % repo_path) from e
-                    input_pattern = os.path.join(git_dir, "**", "*.js")
-                    input_pattern_noisy = os.path.join(git_dir_noisy, "**", "*.js")
-                    model_path = os.path.join(git_dir_noisy, "style-analyzer-model", "model.asdf")
-                else:
-                    input_pattern = os.path.join(repo_path, "**", "*.js")
-                    input_pattern_noisy = os.path.join(repo_path + "_noisy", "**", "*.js")
-                    model_path = os.path.join(repo_path, "model.asdf")
+                log.info("Fetching %s", repo_path)
+                git_dir = os.path.join(tmpdirname, repo)
+                git_dir_noisy = os.path.join(tmpdirname, repo + "_noisy")
+                cmd1 = "git clone --single-branch --branch master %s %s" % (repo_path, git_dir)
```

Comment:
it is a good idea to have a repository with two commits in the `data` directory. 
The improvement is not related to current PR, so just a note.